plutus-core/testlib/PlutusCore/Generators/PIR/GenerateTypes.hs

```diff
@@ -139,6 +150,7 @@ genAtomicType k = do
       lam k1 k2 = do
         x <- genMaybeFreshTyName "a"
         TyLam () x k1 <$> bindTyName x k1 (genAtomicType k2)
+  -- TODO: probably should be 'frequencyTm'?
```

Comment:
Perhaps, but its unclear what the frequencies should be. What do you have in mind?

Reply:
I've [implemented](https://github.com/input-output-hk/plutus/pull/4559/commits/ef9e32825833374f66cd8d5ba8c1fff51ac33e52) proper generation of built-in types and calibrated those frequencies, so that they look kinda sensible and don't cause `prop_unify` to fail with out-of-ticks.